src/resources/i2s_esp32.cc

```diff
@@ -15,7 +15,7 @@
 
 #include "../top.h"
 
-#ifdef TOIT_ESP32
+#if 0 // def TOIT_ESP32
```

Comment:
Intentional?

Reply:
yes.
The I2S uses the ADC underneath. It's not allowed to use the old and new version of the ADC driver. Until the I2S code is updated it won't compile.

There is already a follow-up PR that fixes the I2S. (Otherwise, I would have added more comments).